LocalApp/LocalApplication.java

```diff
@@ -263,7 +260,7 @@ private static void uploadToS3() {
 	}
 
 	private static void initEC2() throws FileNotFoundException, IOException {
-		credentials = new PropertiesCredentials(new FileInputStream("C:\\Users\\Haymi\\Documents\\BGU\\DSP\\rootkey.properties"));
+		credentials = new PropertiesCredentials(new FileInputStream(Util.Const.credentialsFilePath));
```

Comment:
made Util package, with Const class to hold this, and some other strings we share between worker/manager/etc.